config/rhoai/kustomization.yaml

```diff
@@ -44,3 +44,5 @@ patches:
 - path: manager_webhook_patch.yaml
 - path: manager_metrics_patch.yaml
 - path: auth_proxy_service_patch.yaml
+- path: mutating_webhook_patch.yaml
```

Comment:
In the short term, we still want to have those two webhooks removed, but replacing #5 with patches at the level of the `config/rhoai` Kustomize overlay, but adding patches that delete those webhooks from there.

Reply:
Thanks @astefanutti , this has been updated in f464f09c820c16f5b029e39a95abf95de2f472a8